packages/devtools_shared/lib/src/memory/event_sample.dart

```diff
@@ -102,13 +100,13 @@ class ExtensionEvent {
       data == null &&
       customEventName == null;
 
-  final int timestamp;
+  final int? timestamp;
```

Comment:
than these fields except the customEventName would just be final w/o the ?

Reply:
this has to be nullable because of the `empty()` constructor